src/modules/0.0.14/aws_vpc/index.ts

```diff
@@ -419,8 +425,13 @@ export const AwsVpcModule: Module2 = new Module2({
       if (serviceName.includes('s3')) return EndpointGatewayService.S3;
       if (serviceName.includes('dynamodb')) return EndpointGatewayService.DYNAMODB;
     },
-    eqTags: (a: { [key: string]: string }, b: { [key: string]: string }) => Object.is(Object.keys(a ?? {})?.length, Object.keys(b ?? {})?.length) &&
-      Object.keys(a ?? {})?.every(ak => (a ?? {})[ak] === (b ?? {})[ak]),
+    eqTags: (a: { [key: string]: string }, b: { [key: string]: string }) => {
+      // remove possible `updated` key
+      delete a.updated;
+      delete b.updated;
```

Comment:
oh why are you doing this?

Reply:
Agree here. This shouldn't be necessary.